lib/metacat.properties

```diff
@@ -529,31 +414,6 @@ document.sitecode=nceas
 document.accNumSeparator=.
 document.accNumPrefix=autogen
 
-######## Harvester section            #########################################
-
-harvester.connectToMetacat=true
-harvester.delay=0
-harvester.administrator=name@institution.edu
-harvester.logPeriod=90
-harvester.maxHarvests=0
-harvester.period=24
-harvester.smtpServer=localhost
-harvester.GetDocError=Error getting EML document from site,Error
-harvester.GetDocSuccess=Success getting EML document from site,Debug
-harvester.GetHarvestListError=Error getting harvest list from site,Error
-harvester.GetHarvestListSuccess=Success getting harvest list from site,Debug
-harvester.HarvesterStartup=Harvester start up,Info
-harvester.HarvesterShutdown=Harvester shut down,Info
-harvester.InsertDocError=Error inserting EML document to Metacat,Error
-harvester.InsertDocSuccess=Success inserting EML document to Metacat,Info
-harvester.MetacatHasDoc=Metacat already has this EML document,Info
-harvester.UpdateDocError=Error updating EML document to Metacat,Error
-harvester.UpdateDocSuccess=Success updating EML document to Metacat,Info
-harvester.ValidateDocError=Error validating EML docoument,Error
-harvester.ValidateDocSuccess=Success validating EML document,Debug
-harvester.ValidateHarvestListError=Error validating harvest list,Error
-harvester.ValidateHarvestListSuccess=Success validating harvest list,Debug
```

Comment:
harvester docs need to reflect deletions: https://github.com/NCEAS/metacat/blob/65cbef21d2ab036c30d3333c3160353148fa7cbc/docs/user/metacat/source/harvester.rst#L50

Reply:
Actually, the harvester part in documentation should be removed and say " we don't  support it any more". I added the comment about the harvester a while ago in the documentation ticket -https://github.com/NCEAS/metacat/issues/1678#issuecomment-1749555872

Reply:
ok - thanks